index.js

```diff
@@ -52,6 +52,15 @@ function getOptions(addonContext) {
     }
   }
 
+  // add babel-plugins from the registry to the options.plugins
+  var pluginWrappers = addonContext.registry.load('babel-plugin');
```

Comment:
Would something like currently used to get [the options](https://github.com/babel/ember-cli-babel/blob/5bde7459ef00811a13cf64c28660bd0f33f81a34/index.js#L33) be ok?

```
var registry = (addonContext.parent && addonContext.parent.registry) || (addonContext.app && addonContext.app.registry);
var pluginWrappers = registry.load('babel-plugin');
...
```